src/main/resources/reference.conf

```diff
@@ -70,13 +70,13 @@ batchedFetches {
 
   #Used to specify the max-time allowed for each block-page to finish processing
   #Currently takes into account the time to process all corresponding accounts for each page
-  blockPageProcessingTimeout = 15 minutes
+  blockPageProcessingTimeout = 1 hour
```

Comment:
Isn't this a little dangerous? Are we assured consistently logging is going to happen during this hour?

Reply:
Turns out we do need this after all. Sometimes the checkpoint table can get really clogged up. Took 48 minutes to break the logjam on my local machine.